client/src/components/HomePage.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react"
+import RestaurantTile from "./RestaurantTile"
+import { Link } from "react-router-dom"
+
+const HomePage = (props) => {
+    const [restaurants, setRestaurants] = useState([])
+
+    const getTop3Restaurants = async () => {
+        try {
+            const response = await fetch("/api/v1/restaurants/top3")
+            if (!response.ok) {
+                throw new Error(`${response.status} (${response.statusText})`)
+            }
+            const body = await response.json()
+            setRestaurants(body.restaurants)
+        } catch (error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getTop3Restaurants()
+    }, [])
+
+    // const top3Restaurants = () => {
+    //     let allRestaurants = restaurants
+    //     allRestaurants.sort((a, b) => {
+    //         return b.averageRating - a.averageRating
+    //     })
+
+    //     let top3 = []
+    //     if (allRestaurants.length > 3) {
+    //         for (let i = 0; i < 3; i++) {
+    //             top3.push(allRestaurants[i])
+    //         } 
+    //     } else {
+    //         top3 = allRestaurants
+    //     }
+
+    //     return top3
+    // }
+
+    const showTopThree = restaurants.map((restaurant) => {
+        return (
+            <RestaurantTile
+                key={restaurant.id}
+                restaurant={restaurant}
+            />
+        )
+    })
+
+    return (
+        <div class="page grid-container">
+            <div class="grid-container">
+                <div class="headerText">
+                    <h2 class="page header" align="center">Welcome to the most Radical Review Roadblock</h2>
+
+                    <p align="center" class="text">Hey there fellow x-gamer! Have you ever been to restaurant and thought to yourself "man this is totatally not tubular man!" Well we share your pain dude guy man. We want to bring to you a place where you can totally check out the best places and put the poopy buttholes in their place!</p>
```

Comment:
Im okay with a bit of tongue and cheek, but this will come across as very unprofessional.

no poopy buttholes
and women play games too, so lets adjust the language